src/views/components/tooltip/testid-tooltip/TestIdTooltip.js

```diff
@@ -0,0 +1,55 @@
+import React, { Component } from 'react';
+import { Dimensions } from 'react-native-web';
+import { string, node } from 'prop-types';
+import Tooltip from '../Tooltip';
+
+class TestIdTooltip extends Component {
+  static defaultProps = {
+    id: 'test-id',
+  };
+
+  static propTypes = {
+    id: string,
+    children: node,
+  };
+
+  handleMouseOver = () => {
+    // eslint-disable-next-line no-console
+    console.warn( 'TestId:', this.props.id );
+  };
+
+  render() {
+    const { id, children } = this.props;
+
+    const dimensions = Dimensions.get( 'window' );
+
+    let isDebugMode = window.originalQueryParams.showcodes;
```

Comment:
The unchecked use of `window` here may break Native builds, we should add `typeof window !== 'object` to fix.